src/scss/02_atoms/_accordion.scss

```diff
@@ -91,11 +90,9 @@
   visibility: hidden;
 
   .a-accordion.is-open & {
-    border-bottom: 1px solid color(shape);
-    display: flex;
     flex: 1 0 auto;
     opacity: 1;
-    padding: rem(map-get($accordionConfig, accordionPadding));
+    padding: 0 rem(map-get($accordionConfig, accordionPadding)) rem(map-get($accordionConfig, accordionPadding)) rem(map-get($accordionConfig, accordionPadding));
```

Comment:
Metti `rem(map-get($accordionConfig, accordionPadding))` in una variabile, oppure usa `padding: rem(map-get($accordionConfig, accordionPadding)); padding-top: 0;`